lib/api/statistics.js

```diff
@@ -91,7 +91,7 @@ function getStatistics ({ productIds, environment, key, type, filters = {} }) {
   return Promise.all(
     productIdChunks.map(products => new Promise((resolve, reject) => {
       const requestUri = `${uri}&filter=ProductId:${products.join(',')}`
-      fetch(requestUri)
+      return fetch(requestUri)
```

Comment:
If inline_ratings updates bv-ui-core after this is merged they should make sure to test this change.